app/controllers/api/nodes_controller.rb

```diff
@@ -31,7 +31,12 @@ def show
       response.last_modified = node.timestamp
 
       if node.visible
-        render :xml => node.to_xml.to_s
+        @node = node
```

Comment:
Perhaps `@node` throughout

Reply:
There's a bunch of other places like this too - generally if you find yourself needing to rename `variable` to `@variable` it probably should have been `@variable` from the start. But it's not critical, we can always refactor this later.

Reply:
Right, I also found this somewhat awkward... makes total sense.